components/Avatar/index.d.ts

```diff
@@ -6,7 +6,6 @@ export interface AvatarProps {
     text?: string;
     hasToggle?: boolean;
     href?: string;
-    size?: string;
     onClick?: () => {};
```

Comment:
Here the onClick as prop is declared only in the index file. Is supposed to be like this?

Reply:
Same case of toggle here.